pkg/signature/kms/azure/signer.go

```diff
@@ -87,12 +87,14 @@ func LoadSignerVerifier(defaultCtx context.Context, referenceStr string) (*Signe
 // All other options are ignored if specified.
 func (a *SignerVerifier) SignMessage(message io.Reader, opts ...signature.SignOption) ([]byte, error) {
 	var digest []byte
+	ctx := context.Background()
```

Comment:
Since the `SignerVerifier` is still instantiated with a default context, should we use that as the default instead of `context.Background`?

Reply:
Nevermind, @codysoyland beat me to it